src/Command/DrupalUpdate/DrupalUpdateCommand.php

```diff
@@ -0,0 +1,109 @@
+<?php
+
+namespace Acquia\Cli\Command\DrupalUpdate;
+
+use Acquia\Cli\Command\CommandBase;
+use Acquia\Cli\Exception\AcquiaCliException;
+use Composer\Semver\Comparator;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Output\OutputInterface;
+
+class DrupalUpdateCommand extends  CommandBase{
+  /**
+   * @var string
+   */
+  protected static $defaultName = 'app:update-d7-packages';
+
+  /**
+   * @var mixed|null
+   */
+  private $drupalCoreVersion;
+  /**
+   * @var DrupalPackageUpdate
+   */
+  private DrupalPackageUpdate $drupalPackageUpdate;
+
+  /**
+   * @return DrupalPackageUpdate
+   */
+  public function getDrupalPackageUpdate(): DrupalPackageUpdate {
+    return $this->drupalPackageUpdate;
+  }
+
+  /**
+   * @param DrupalPackageUpdate $drupalPackageUpdate
+   */
+  public function setDrupalPackageUpdate(DrupalPackageUpdate $drupalPackageUpdate): void {
+    $this->drupalPackageUpdate = $drupalPackageUpdate;
+  }
+
+  /**
+   * @return mixed
+   */
+  public function getDrupalCoreVersion() {
+    return $this->drupalCoreVersion;
+  }
+
+  /**
+   * @param mixed $drupalCoreVersion
+   */
+  public function setDrupalCoreVersion($drupalCoreVersion): void {
```

Comment:
Missing both function's return type declaration and parameter's type declaration